src/WebJobs.Script/Host/BlobLeaseManager.cs

```diff
@@ -13,26 +13,28 @@ namespace Microsoft.Azure.WebJobs.Script
 {
     internal sealed class BlobLeaseManager : IDisposable
     {
-        private const string LockBlobName = "hostlock";
-        private const string HostContainerName = "azure-functions-host";
+        internal const string LockBlobName = "hostlock";
```

Comment:
Wondering if you shouldn't just name the blob "Host"? If you put it under /locks (comment below) that would make sense. Also, our other locks look like "locks/functions-test/Host.Functions.TimerTriggerCSharp1.Listener" - function level locks start with "Host.Functions.xxx".